themes/classic/_dev/css/partials/_helpers.scss

```diff
@@ -47,3 +47,19 @@
 ._shadow{
   @include box-shadow;
 }
+
+._pull-right {
+  float:right;
+}
+
+._align-center {
```

Comment:
Yes, please don't create helpers for that. Some Bootstrap 4 nomenclature has changed. Use pull-xs-right and text-center instead. See the migration page: http://v4-alpha.getbootstrap.com/migration/